sections/04_study.md

```diff
@@ -478,114 +478,103 @@ within neural networks.
+@doi:10.1109/TCBB.2014.2343960 @doi:10.1038/srep11476] made important technical
+advances in adapting deep learning to protein structure prediction, but were
+unable to achieve significant improvement over the industry standard PSIPRED
+[@doi:10.1006/jmbi.1999.3091], which uses two shallow feedforward neural
+networks. In 2014, Zhou and Troyanskaya demonstrated that they could improve Q8
+accuracy by using a deep supervised and convolutional generative stochastic
+network [@arxiv:1403.1347]. In 2016 Wang et al. developed a deep CNF
+model (DeepCNF) that significantly improved Q3 and Q8 accuracy as well as
+prediction of solvent accessibility and disorder regions [@doi:10.1038/srep18962
+@doi:10.1007/978-3-319-46227-1_1]. DeepCNF was the first tool to achieve Q3
+accuracy of 84-85%, much higher than the 80% accuracy maintained by PSIPRED for
+more than 10 years. This improvement may be mainly due to the ability of
+convolutional neural fields to capture long-range sequential information, which
+is important for beta strand prediction. Nevertheless, improving secondary
+structure prediction from 80% to 84-85% is unlikely to result in a commensurate
+improvement in tertiary structure prediction since secondary structure mainly
+reflects coarse-grained local conformation of a protein structure.
+
+Protein contact prediction and contact-assisted folding (i.e. folding proteins
+using predicted contacts as restraints) represents a promising new direction for
+ab initio folding of proteins without good templates in PDB. Evolutionary
+coupling analysis (ECA) is effective for proteins with a very large number
+(>1000) of sequence homologs [@doi:10.1371/journal.pone.0028766], but fares
+poorly for proteins without many sequence homologs. By combining ECA with a few
+other protein features, shallow neural network-based methods such as MetaPSICOV
+[@doi:10.1093/bioinformatics/btu791] and CoinDCA-NN
+[@doi:10.1093/bioinformatics/btv472] have shown some advantage over ECA for
+proteins with few sequence homologs, but their accuracy is far from optimal. In
+recent years, various architectures have been explored for contact prediction.
+For example, Di Lena et al. introduced a deep spatio-temporal neural network (up
+to 100 layers) that utilizes both spatial and temporal features to predict
+protein contacts[@doi:10.1093/bioinformatics/bts475]. Eickholt and Cheng
+combined deep belief networks and boosting techniques to predict protein
+contacts [@doi:10.1093/bioinformatics/bts598] and trained deep networks by
+layer-wise unsupervised learning followed by fine-tuning of the entire network.
+Skwark and Elofsson et al. developed an iterative deep learning technique for
+contact prediction by stacking a series of Random Forests
+[@doi:10.1371/journal.pcbi.1003889]. However, blindly tested in the well-known
+CASP competitions, these methods did not show any advantage over MetaPSICOV
+[@doi:10.1093/bioinformatics/btu791], a method using two cascaded neural
```

Comment:
Cut ", a method using two cascaded neural networks".  We describe MetaPSICOV above.